src/main/java/io/supertokens/webserver/InputParser.java

```diff
@@ -137,7 +137,11 @@ public static String parseStringOrThrowError(JsonObject element, String fieldNam
             if (!stringified.contains("\"")) {
                 throw new Exception();
             }
-            return ((JsonObject) element).get(fieldName).getAsString();
+            String s = element.get(fieldName).getAsString().trim();
+            if (s.contains("@")) {
+                s = s.toLowerCase();
+            }
+            return s;
         } catch (Exception e) {
```

Comment:
Also, please add tests for this change please :)

Reply:
Done